thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/alertFilterAutotune/AlertFilterAutotuneFactory.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.thirdeye.anomalydetection.alertFilterAutotune;
```

Comment:
I did not see a reason why we need this AlertFilterAutotuneFactory. Thirdeye only call autotune from anomaly-detection lib, how we tune it is determined in anomaly-detection lib. On thirdeye, we at most need a interface and a dummyAutoTune as the base class. We do not need a factory.